inc/saiswitch.h

```diff
@@ -2851,13 +2864,13 @@ typedef enum _sai_switch_attr_t
      *
      * true: Trigger the switch isolate process
      * false: Undo the isolation operation.
-     * This attribute is for Fabric Chassis only
+     * This attribute is for Fabric or VOQ Chassis only. This attribute is set as an action to the isolation status change notification callback. Setting this flag to true means that there is no data traffic flowing through the fabric ports. Fabric control traffic like credit grants and control messages will still continue to flow.
```

Comment:
> This attribute is set as an action to the isolation status change notification callback.

That's not the only use case. Switch can be isolated for operational reasons as well. The ask from meeting on 05/04 was to define what happens in response to set/unset of this attribute. That part is covered by

"Setting this flag to true means that there is no data traffic flowing through the fabric ports. Fabric control traffic like credit grants and control messages will still continue to flow."

Consider wording it as

"Setting this flag to true will stop data traffic from flowing on local fabric ports. Fabric control traffic like credit grants and control messages will still continue to flow. Setting this flag to false will (re)enable data traffic on fabric ports."

Reply:
Updated the comments